mesos_master/README.md

```diff
@@ -12,26 +12,25 @@ This check collects metrics from Mesos masters for:
 And many more.
 ## Setup
 ### Installation
-The installation is the same on Mesos with and without DC/OS. 
-Run the docker-dd-agent container on each of your Mesos master nodes:
+The installation is the same on Mesos with and without DC/OS.
+Run the datadog-agent container on each of your Mesos master nodes:
 
 ```
-docker run -d --name dd-agent \
+docker run -d --name datadog-agent \
   -v /var/run/docker.sock:/var/run/docker.sock:ro \
   -v /proc/:/host/proc/:ro \
   -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
-  -e API_KEY=<YOUR_DATADOG_API_KEY> \
+  -e DD_API_KEY=<YOUR_DATADOG_API_KEY> \
   -e MESOS_MASTER=yes \
   -e MARATHON_URL=http://leader.mesos:8080 \
-  -e SD_BACKEND=docker \
-  datadog/docker-dd-agent:latest
+  datadog/agent:latest
 ```
 
 Substitute your Datadog API key and Mesos Master's API URL into the command above.
 
 ### Configuration
 
-If you passed the correct Master URL when starting docker-dd-agent, the Agent is already using a default `mesos_master.yaml` to collect metrics from your masters; you don't need to configure anything else. See the [sample mesos_master.yaml](https://github.com/DataDog/integrations-core/blob/master/mesos_master/conf.yaml.example) for all available configuration options.
+If you passed the correct Master URL when starting datadog-agent, the Agent is already using a default `mesos_master.yaml` to collect metrics from your masters; you don't need to configure anything else. See the [sample mesos_master.yaml](https://github.com/DataDog/integrations-core/blob/master/mesos_master/conf.yaml.example) for all available configuration options.
```

Comment:
Shall we update the config path to the new `mesos_master.d/conf.yaml.default` ?
BTW, it's not obvious to everyone that they don't need to remove it if they do add a `mesos_master.d/conf.yaml` or `mesos_master.conf` in `/conf.d` (the default one is then overriden). Could that fit in that section?

Reply:
what do you recommend @l0k0ms ?

Reply:
After speaking with @masci it has been decided to enforce A6 notation everywhere, so we should here update the config path, correct text should then be: 

If you passed the correct Master URL when starting datadog-agent, the Agent is already using a default `mesos_master.d/conf.yaml` to collect metrics from your masters; you don't need to configure anything else. See the [sample mesos_master.d/conf.yaml.default](https://github.com/DataDog/integrations-core/blob/master/mesos_master/conf.yaml.example) for all available configuration options.